projects/components/src/multi-select/multi-select.component.scss

```diff
@@ -54,7 +54,6 @@
 .multi-select-content {
   @include dropdown();
   min-width: 120px;
-  max-height: 204px;
```

Comment:
Shouldn't it have some max height? I'd rather it scroll then run off the screen.

Reply:
How do we choose this value? Different use cases may show different set of options. Do you want to expose an input for this?

Reply:
Or may be we can set a max height which is equal to available height, that is, from trigger element's position to the bottom on the window.

Reply:
Let me create a separate story for this change.